chrysom/client.go

```diff
@@ -386,7 +402,7 @@ func (c *Client) Start(ctx context.Context) error {
 				return
 			case <-c.observer.ticker.C:
 				outcome := SuccessOutcome
-				items, err := c.GetItems("")
+				items, err := c.GetItems("",context.TODO())
```

Comment:
`otel` will just start a new trace in this case, right?

Reply:
Yes, `otel` will start the new Trace in this case if TraceMiddleware is in place. Here that is not in place.